indico/modules/events/editing/client/js/editing/EditableSubmissionButton.jsx

```diff
@@ -97,9 +107,11 @@ export default function EditableSubmissionButton({
       </FinalForm>
       {editableTypes.length === 1 ? (
         <Button onClick={() => setCurrentType(editableTypes[0])} primary>
-          <Translate>
-            Submit <Param name="editableType" value={editableTypes[0]} />
-          </Translate>
+          {text || (
+            <Translate>
+              Submit <Param name="editableType" value={editableTypes[0]} />
```

Comment:
I know you just reindented it, but this is not properly translatable. it should use a lookup dict with static messages for the different types or simply 3 separate conditions like we do for the `<Modal.Header>` in the same file.